Common/src/main/java/at/petrak/hexcasting/api/casting/mishaps/MishapUnenlightened.kt

```diff
@@ -18,14 +19,15 @@ class MishapUnenlightened : Mishap() {
 
     override fun execute(env: CastingEnvironment, errorCtx: Context, stack: MutableList<Iota>) {
         env.mishapEnvironment.dropHeldItems()
-        env.caster?.sendSystemMessage("hexcasting.message.cant_great_spell".asTranslatedComponent, true)
```

Comment:
why was the `true` removed here?

Reply:
because it is not a argument on the LivingEntity version

Reply:
(that last boolean only exists on the player version)
ofc it should also move to the Environment method to send message. but thats a future PR)